src/responses/saved-collections.feed.response.ts

```diff
@@ -0,0 +1,70 @@
+export interface SavedCollectionsFeedResponseRootObject {
+  items: SavedCollectionsFeedResponse[];
+  num_results: number;
+  more_available: boolean;
+  auto_load_more_enabled: boolean;
+  status: string;
+  next_max_id: string;
+}
+
+export interface SavedCollectionsFeedResponse {
+  collection_id: string;
+  collection_name: string;
+  collection_type: string;
+  cover_media: CoverMedia;
+  cover_media_list: CoverMediaList[];
+  collection_media_count: number;
+  viewer_access_level: string;
+}
+
+export enum SavedCollectionCollectionType {
```

Comment:
Shared collections type is missing. Any idea how to determine it?

Reply:
There are no need in addition collection type. Instead the constant `FACEBOOK_ANALYTICS_APPLICATION_ID ` should be changed to the newer value `936619743392459`.